source/fundamentals/builders/sort.txt

```diff
@@ -11,3 +11,326 @@ Sorts Builders
+fields with identical values. This means that if you pass "letter" to
+the ``ascending()`` method and use the returned sort criteria to sort
+the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`, the first
+document returned could be any of the following documents:
+
+.. code-block:: json
+
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+
+If you need guaranteed sort order for documents that
+have fields with identical values, you can specify additional fields to sort
+on in the event of a tie.
+
+We can specify an ascending sort on the ``letter`` field followed by the
+``_id`` field as follows:
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.ascending;
+
+   // <MongoCollection setup code here>
+   
+   collection.find().sort(ascending("letter", "_id"));
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`  
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+
+Combining Sort Criteria
+-----------------------
+
+To combine sort criteria, use the ``Sorts.orderBy()`` static factory
+method. The ``orderBy()`` method builds sort criteria that apply passed 
+in sort criteria from left to right in the event of ties. 
+
+In the following code snippet, we use the ``orderBy()`` method to combine a
+descending sort on the ``letter`` field with an ascending sort on the
+``_id`` field.
+
+.. code-block:: java
+
+   import static com.mongodb.client.model.Sorts.orderBy;
+   import static com.mongodb.client.model.Sorts.ascending;
+   import static com.mongodb.client.model.Sorts.descending;
+
+   // <MongoCollection setup code here>
+
+   Bson orderBySort = orderBy(descending("letter"), ascending("_id"));
+   collection.find().sort(orderBySort);
+
+The code snippet above returns the documents in the
+:ref:`sort_example collection <sorts-builders-sort-example>`
+in the following order: 
+
+.. code-block:: json
+
+   {"_id": 1, "letter": "c", "food": "coffee with milk"}
+   {"_id": 6, "letter": "c", "food": "maple donut"}
+   {"_id": 4, "letter": "b", "food": "coffee with sugar"}
+   {"_id": 2, "letter": "a", "food": "donuts and coffee"}
+   {"_id": 3, "letter": "a", "food": "maple syrup"}
+   {"_id": 5, "letter": "a", "food": "milk and cookies"}
+
+Text Search
+-----------
+
+If you perform a text search on your collection with the
+:manual:`$text </reference/operator/query/text/>` query operator, you 
+can sort your results by how well they match your ``$search`` String using
```

Comment:
Thanks for the note. I have rewritten the paragraph to not reference `$search`, and to start with the idea of sorting, I.E "you can specify the order..." as first sentence. Fix: https://github.com/mongodb/docs-java/pull/59/commits/eafd2710118848f7ff8354aa13e288dee0098e0b

Going forward, I will try to avoid burying the lead on sections, and will try to avoid introducing a large number of terms at the same time.